cloud/terraform/terraform_controller.py

```diff
@@ -0,0 +1,105 @@
+from ast import alias
+import os
+import json
+
+
+class TerraformController:
+    def __init__(self, cloud):
```

Comment:
I think we should be able to pass a configuration file. It will probably be easier to manage in the future.

Then, depending on the cloud provider of the config received, we set some attributes that will be needed by the rest of the code later. Here we can introduce a class or a library that will validate that the config received is consistent (e.g. from a YAML, a json or even a .config file).

Maybe not to be implemented now, but as we will need more and more customization for the providers (the aws_profile is one example right now), we will have to do something to digest that config from somewhere.